Spec/Auth.swift

```diff
@@ -3,17 +3,113 @@ import Ably.Private
 import Nimble
 import Quick
 import Aspects
+        
+        private var testHTTPExecutor: TestProxyHTTPExecutor!
+                private func testOptionsGiveDefaultAuthMethod(_ caseSetter: (ARTAuthOptions) -> Void) {
+                    let options = ARTClientOptions()
+                    caseSetter(options)
+                    
+                    let client = ARTRest(options: options)
+                    
+                    expect(client.auth.internal.method).to(equal(ARTAuthMethod.token))
+                }
+                // Cases:
```

Comment:
If the spacing here bothers people, I can update the migrator to add some newlines between declarations.